### ISSUE TITLE
How to reveal full <app-header> immediately on a condenses+reveals+fixed attributes when slide back up without scrolling to top

### ISSUE THREAD

**Issue by andrewspy** [PolymerElements/app-layout#232]
Referring to the following code:-

```html
 <app-header-layout has-scrolling-region>

    <app-header condenses reveals fixed effects="waterfall">
      <app-toolbar style="height: 48px;">
        <paper-icon-button icon="menu" drawer-toggle></paper-icon-button>
        <div title>My App</div>
      </app-toolbar>
      <app-toolbar primary style="height: 48px;">
        <paper-tabs attr-for-selected="name" fallback-selection="tab-1" selected={{selected}} style="width: 100%" noink scrollable>
          <paper-tab name="tab-1">Tab 1</paper-tab>
          <paper-tab name="tab-2">Tab 2</paper-tab>
        </paper-tabs>
      </app-toolbar>
    </app-header>

    <iron-pages role="main" selected="[[page]]" attr-for-selected="name">
      <my-view1 name="view1"></my-view1>
      <my-view2 name="view2"></my-view2>
      <my-view3 name="view3"></my-view3>
    </iron-pages>

  </app-header-layout>
```
I intend to fixed the `<paper-tabs>`, and reveal the first `<app-toolbar>` immediately when slide back up. Currently I have to scroll to top for the first `<app-toolbar>` to reveal itself.

I want to know if current `<app-header>` implement this, or is there a plan to implement this in the future. Thanks!

**Comment by blasten** [comment]
@andrewspy in that case you can do this:

```html
<app-header condenses reveals fixed effects="waterfall">
  <div primary>
     toolbars
  </div>
</app-header>
```

Feel free to reopen if you find an issue.

**Comment by blasten** [comment]
@andrewspy This issue is fixed now and tagged as `0.9.1`, your code should just work now.

**Comment by andrewspy** [comment]
@blasten Confirmed that the issue has been fixed in `0.9.1`.

Thanks for the quick response!
